src/Checker.php

```diff
@@ -7,8 +7,14 @@ class Checker {
 	/**
 	 * Check that Git, Composer, and Hub are available on the filesystem.
 	 */
-	public static function check_executables() {
-		$execs = array( 'git', 'composer', 'hub' );
+	public static function check_executables($should_check_gitlab) {
```

Comment:
Instead of `$should_check_gitlab` as a parameter what do you think about passing `$git_provider` and using a `switch` below. The default can be `github` with a `case` for GitLab. This will allow easier extension for more providers in the future.